pages/1.9/security/ent/secrets/seal-store/index.md

```diff
@@ -12,15 +12,29 @@ You may want to manually seal the Secret Store to protect its contents from an i
 
 Sealed Secret Stores cannot be accessed from the GUI. Secret values cannot be retrieved using the [Secrets API](/1.9/security/ent/secrets/secrets-api/). Services that depend on values in the Secret Store may fail to deploy.
 
-To seal the Secret Store, complete the following steps.
+To seal the Secret Store, complete the following steps. They will seal just a single instance of dcos-secrets. If the cluster URL obtained through `dcos config show core.dcos_url` points to a load balancer and there is more than one master instance in the cluster, then these steps should be issued against each and every master node instead and the cluster URL should be changed to the address of individual master nodes.
 
-**Prerequisites:** 
+**Prerequisites:**
 
 - You must have the [DC/OS CLI installed](/1.9/cli/install/) and be logged in as a superuser via `dcos auth login`.
 
 - If your [security mode](/1.9/security/ent/#security-modes) is `permissive` or `strict`, you must [get the root cert](/1.9/networking/tls-ssl/get-cert/) before issuing the curl commands in this section.  If your [security mode](/1.9/security/ent/#security-modes) is `disabled`, you must delete `--cacert dcos-ca.crt` from the commands before issuing them.
 
 
+1. From a terminal prompt, check the status of the Secret Store via the following command.
+
+   ```bash
+   curl --cacert dcos-ca.crt -H "Authorization: token=$(dcos config show core.dcos_acs_token)" $(dcos config show core.dcos_url)/secrets/v1/seal-status/default
+   ```
+
+1. The Secret Store service should return the following response.
```

Comment:
`s/return the following response/return a response like the following` or similar.

That is - this indicates that the response will be exactly like this. Then the next paragraph says "it might be slightly different".